### PR TITLE
Add alternate build files into the debug zip.

### PR DESCRIPTION
**Pull request by rseanhall**
- This makes the solutions actually load.
- This makes it clear that the projects aren't meant to be built.

### CODE REVIEW

**Comment by robmen** [comment]
This is a great idea but I don't like the name of the files. `decoy` feels very misleading. I'd rather call them "ForDebugZip" or something more direct as to their purpose.

File: src/Setup/Zip/decoy/WixBuild.props [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  <copyright file="WixBuild.props.decoy" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
+<Project InitialTargets="CheckRequiredProperties" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0" />
```
Comment (robmen) [inline src/Setup/Zip/decoy/WixBuild.props]: I think a comment as the body of this project that describes what this file is for would be a good idea when anyone trips across it.

File: src/Setup/Zip/decoy/wix.proj [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  <copyright file="wix.proj.decoy" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
+<Project InitialTargets="CheckRequiredProperties" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0" />
```
Comment (robmen) [inline src/Setup/Zip/decoy/wix.proj]: Same as above: I think a comment as the body of this project that describes what this file is for would be a good idea when anyone trips across it.

File: src/Setup/Zip/decoy/WixBuild.targets [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
+<Project InitialTargets="CheckRequiredProperties" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+  <Target Name="CheckRequiredProperties">
+    <Error
+      Code="WIXDECOY000"
```
Comment (robmen) [inline src/Setup/Zip/decoy/WixBuild.targets]: Not big on `DECOY` here but this is a brilliant idea.

**Comment by rseanhall** [comment]
I amended the commit and force pushed to remove any references to "decoy" and added comments to the empty projects.

**Pull request closed without merging**